src/opera/util/h5_utils.py

```diff
@@ -340,8 +53,8 @@ def get_hdf5_group_as_dict(file_name, group_path, ignore_keys=None):
         python dict containing variable data from the group path location.
     """
     group_dict = {}
-    with h5py.File(file_name, 'r') as hf:
-        group_object = hf.get(group_path)
+    with h5py.File(file_name, 'r') as hf:    # pylint: disable=invalid-name
```

Comment:
Rather than disable this finding, I would rather actually address it and use a better name than `hf` here. Even `h5file` would be better.

Reply:
I think h5file is very clear and a good choice.